apps/public-reference/pages/create-account.tsx

```diff
@@ -1,21 +1,29 @@
+import { useContext } from "react"
 import { useForm } from "react-hook-form"
-import axios from "axios"
-import { Button, Field, FormCard, Icon, LinkButton } from "@bloom-housing/ui-components"
+import {
+  Button,
+  Field,
+  FormCard,
+  Icon,
+  LinkButton,
+  UserContext,
+} from "@bloom-housing/ui-components"
 import FormsLayout from "../layouts/forms"
 import { emailRegex } from "../lib/emailRegex"
 
 export default () => {
+  const { register: registerUser } = useContext(UserContext)
```

Comment:
I know we have the register vs. register conflict here, but I'm wondering if we can rename the original function name so it doesn't have to get rewritten?